src/dmd/link.d

```diff
@@ -20,7 +20,6 @@ import core.sys.posix.stdlib;
 import core.sys.posix.unistd;
 import core.sys.windows.winbase;
 import core.sys.windows.windef;
-import core.sys.windows.winreg;
```

Comment:
Are winbase and windef still needed?

Reply:
> Are winbase and windef still needed?

Yes, winbase: STARTUPINFO, winbase: BOOL, DWORD